src/TicketContext.tsx

```diff
@@ -37,15 +47,18 @@ type TicketReducer = (
   action: TicketReducerAction,
 ) => TicketReducerState;
 
-const ticketReducer: TicketReducer = (prevState, action) => {
+const ticketReducer: TicketReducer = (
+  prevState,
+  action,
+): TicketReducerState => {
```

Comment:
Is typing `TicketReducerState` necessary here when the variable is already typed to `TicketReducer` (which in turn is already typed to return `TicketReducerState`)?